src/vivarium_gates_nutrition_optimization/data/extra_gbd.py

```diff
@@ -1,65 +1,65 @@
 from gbd_mapping import sequelae
 from vivarium_gbd_access import constants as gbd_constants
-from vivarium_gbd_access import gbd
 from vivarium_gbd_access import utilities as vi_utils
```

Comment:
I think it is confusing to call vivarium_gbd_access.utilities vi_utils; let's discuss these coding conventions soon.